client/src/components/LeafletOsmMap/SavedPlaces.js

```diff
@@ -0,0 +1,159 @@
+import {h, Component} from "preact";
+import {makeRequest} from "../../js/server-requests-utils";
+
+/**
+ * Leaflet related imports: leaflet, pouchdb module, and routing machine module
+ */
+import '../../../node_modules/leaflet/dist/leaflet.css';
```

Comment:
this and the next `.css` file is not needed in this module since it doesn't actually do any of the rendering.

Reply:
All Leaflet imports were removed from the `.js` utility file